README.md

```diff
@@ -34,22 +34,27 @@ NOTE: if you will only be using Python RDK, you can skip this section and jump t
         sudo apt install build-essential git cmake cmake-qt-gui -y
 
 2. Choose a directory for installing ``flexiv_rdk`` library and all its dependencies. For example, a new folder named ``rdk_install`` under the home directory.
-3. In a new Terminal, use CMake to configure ``flexiv_rdk``:
+3. In a new Terminal, run the provided script to compile and install all dependencies to the installation directory chosen in step 1:
+
+        cd flexiv_rdk/thirdparty
+        bash build_and_install_dependencies.sh ~/rdk_install
+
+4. In a new Terminal, use CMake to configure ``flexiv_rdk``:
 
         cd flexiv_rdk
         mkdir build && cd build
         cmake .. -DCMAKE_INSTALL_PREFIX=~/rdk_install
 
    NOTE: ``-D`` followed by ``CMAKE_INSTALL_PREFIX`` is a CMake parameter specifying the path of the chosen installation directory. Alternatively, this configuration step can also be done through CMake GUI.
```

Comment:
Why not recommend users to use a GUI for this step? Errors often occur in users who are not very familiar with CMake, and for advanced users, if there are certain special scenarios (such as when remote login cannot open GUI, they naturally know the command line approach)

Reply:
GUI tends to complicate things for beginners, providing only terminal commands is more straightforward as it's copy-pasteable.